lib/modules/manager/helmfile/artifacts.ts

```diff
@@ -59,9 +69,42 @@ export async function updateArtifacts({
         constraint: config.constraints?.kustomize,
       });
     }
-    await exec(`helmfile deps -f ${quote(packageFileName)}`, {
+
+    const cmd: string[] = [];
+    const doc = yaml.load(newPackageFileContent) as Doc; //TODO #9610
```

Comment:
this should be validated with `zod` schema

Reply:
You can use this as an example: https://github.com/renovatebot/renovate/blob/0a4be4a989788fa7f920b1ff8bff3326970aea53/lib/modules/manager/pep621/utils.ts#L111

The process will be the same for TOML and YAML.